### ISSUE TITLE
Toggle Evaluation Coverage fails

### ISSUE THREAD

**Issue by timothyhinrichs** [open-policy-agent/vscode-opa#41]
The Coverage option no longer highlights the lines participating in the evaluation.

To reproduce, open a new file, add the following policy, highlight `allow`, and run `OPA: Toggle Evaluation Coverage`.  The results of evaluation get displayed in the usual Output pane, but none of the lines get highlighted.

```
package foo
allow {
	true
}
allow {
	false
}
```

**Comment by tsandall** [comment]
Dug into this a little bit and found something odd...leaving a note here so that it's not lost.

When the extension runs `opa eval` to generate the coverage report, the subprocess looks like this:

```
echo 'allow' | opa eval --coverage --stdin --bundle=file:///home/torin/scratch/vscode-exp --package foo --metrics
```

The generated coverage report will be something like:

```
  "coverage": {
    "files": {
      "/home/torin/scratch/vscode-exp/file:/home/torin/scratch/vscode-exp/x.rego": {},
      "file:/home/torin/scratch/vscode-exp/x.rego": {
        "covered": [
          {
            "start": {
              "row": 3
            },
            "end": {
              "row": 6
            }
          }
        ],
        "coverage": 100
      }
    },
    "coverage": 100
  }
```

There are two issues here...

1. The first key `"/home/torin/scratch/vscode-exp/file:/home/torin/scratch/vscode-exp/x.rego"` is wrong and it's coming from the internal file loading package inside of OPA proper. Something is broken there and needs to be debugged.
2. The endsWith check on [L156](https://github.com/open-policy-agent/vscode-opa/blob/a73d5d96ccbab2e8c24d4f320e05548cf2825950/src/extension.ts#L156) of the extension will fail for the second key because it's looking at the document filename (which will be something like `/home/torin/scratch/.../x.rego` and the latter is not ending with the file:/ URI.)

The second issue (and the one that's preventing the painting) can be fixed by stripping the URI scheme from the coverage keys.

**Comment by mykter** [comment]
It's also not obvious to me whether coverage is meant to be on or not - "toggle" is fine if it's clear what your current state is but if there's no indicator then you just have to hit it twice!

My failure mode is a bit different to the OP's - when I hit toggle it clears the output pane. The 'Test Workspace' command works (the output pane shows all of the tests).

**Comment by tsandall** [comment]
This has been fixed as of OPA v0.29
